apps/homework/models.py

```diff
@@ -103,25 +119,45 @@ def get_remote_submission_info(self):
                 os.environ.get('CODALAB_SUBMISSION_PASSWORD')
             )
         )
+        print('score_api_url',score_api_url)
+        print(score_api_resp.content)
         if score_api_resp.status_code == 200:
             data = score_api_resp.json()
             if data.get('status'):
                 print("Data found for submission. Returning scores.")
```

Comment:
It might be nicer to set default values for `stored_status` and `stored_score` (None) then on each condition set them to their correct value. I think this will save some space, and then we can call `self.save()` at the end instead of in each condition.

Reply:
Good call. Done.